src/views/catalog/CreateFromInstanceTypes/components/VMDetailsSection/VMDetailsSection.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { Dispatch, SetStateAction, useState } from 'react';
+
+import { V1beta1DataSource } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
+import { t } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { ExpandableSection } from '@patternfly/react-core';
+
+import { InstanceTypeState } from '../../utils/constants';
+
+import VMDetailsBody from './components/VMDetailsBody/VMDetailsBody';
+
+import './VMDetailsSection.scss';
+
+type VMDetailsSectionProps = {
+  vmName: string;
+  setVMName: Dispatch<SetStateAction<string>>;
+  namespace: string;
+  bootSource: V1beta1DataSource;
+  instancetype: InstanceTypeState;
+};
+
+const VMDetailsSection: React.FC<VMDetailsSectionProps> = ({
+  vmName,
+  setVMName,
+  bootSource,
+  instancetype,
+  namespace,
+}) => {
+  const [advancedSectionOpen, setAdvancedSectionOpen] = useState<boolean>(false);
+
+  return (
+    <div className="instancetypes-vm-details-section">
+      <ExpandableSection
+        toggleText={t('Advanced')}
+        data-test-id="expandable-advanced-section"
+        onToggle={() => setAdvancedSectionOpen(!advancedSectionOpen)}
+        isExpanded={advancedSectionOpen}
+        isIndented
+      >
+        Placeholder for SSH section
```

Comment:
I am not sure about this. Aren't we missing translations here? 
`{t('Placeholder for SSH section')}` ?

I expect to see some other text in here later (maybe this is WIP), but I'd use our `t`  function, if that's gonna be visible for the user (looks like it is gonna) .

Reply:
This is just a placeholder. I'm going to open another PR later today to replace that with the actual SSH component.